src/components/tree-item/tree-item.tsx

```diff
@@ -90,14 +90,6 @@ export class TreeItem implements ConditionalSlotComponent, InteractiveComponent
    */
   @Prop({ reflect: true, mutable: true }) lines: boolean;
 
-  /**
-   * Displays checkboxes (set on parent).
-   *
-   * @internal
-   * @deprecated Use `selectionMode="ancestors"` for checkbox input.
-   */
-  @Prop() inputEnabled: boolean;
```

Comment:
Looks like this was already a breaking change at some point since it isn't used anywhere.